app/handlers/handlers_services.py

```diff
@@ -140,7 +107,11 @@ def service_to_twingate_resource(service_body: Body, namespace: str) -> dict:
                     if spec["type"] == ServiceType.LOAD_BALANCER
                     else f"{service_name}.{namespace}.svc.cluster.local"
                 ),
-                "certificateAuthorityCert": get_ca_cert(tls_secret),
+                "certificateAuthorityCert": base64.b64encode(
+                    ResourceProxy.read_certificate_authority_cert_from_secret(
+                        secret
+                    ).encode()
+                ).decode(),
```

Comment:
This is conversion is temporary, it'll be removed in this [PR](https://github.com/Twingate/kubernetes-operator/pull/808/files#diff-8703dce0ccbc9d51ff51eeaab1da0346af9682f282ff0ebd6f275d96e84007b9L110-L114)